src/frontend/src/flows/manage.ts

```diff
@@ -26,6 +28,9 @@ const pageContent = (userNumber: bigint) => html`<style>
     </p>
     <label>User Number</label>
     <div class="highlightBox">${userNumber}</div>
+    <button id="addAdditionalDevice" type="button">
+      Add an additional device
+    </button>
```

Comment:
I would have intuitively put this at the end of the list (mentally, I’d look at the list, notice what’s there, and after reading it might think there is something missing, and then be glad to see the button there). Up to you, though!

Reply:
I wanted to make sure the button always was in view, even if the list got long enough to cause vertical scroll.